src/common/utils/auth.ts

```diff
@@ -1,13 +1,14 @@
+import AppConfig from '../../app/AppConfig';
 import authService from '../../features/auth/authService';
 
 export const getProfileToken = () => {
-  const profileScope = process.env.REACT_APP_TUNNISTAMO_SCOPE_PROFILE;
+  const profileScope = AppConfig.oidcProfileApiClientId;
   const profileToken = JSON.parse(authService.getTokens() as string)[profileScope as string];
   return profileToken;
 };
 
 export const getBerthToken = () => {
-  const berthScope = process.env.REACT_APP_TUNNISTAMO_SCOPE_BERTHS;
+  const berthScope = AppConfig.oidcBerthApiClientId;
   const berthToken = JSON.parse(authService.getTokens() as string)[berthScope as string];
   return berthToken;
 };
```

Comment:
The difference between the scope/clientId names got me thinking. How about a little of refactoring at the same time as getting rid of that difference? For example:
```typescript
/** Get token for the given client. */
const getClientToken = (clientId: string) => {
  return JSON.parse(authService.getTokens() ?? '')[clientId];
};

export const getProfileToken = () => {
  return getClientToken(AppConfig.oidcProfileApiClientId);
};

export const getBerthToken = () => {
  return getClientToken(AppConfig.oidcBerthApiClientId);
};
```